GeometryCore/src/Polygon.cpp

```diff
@@ -27,8 +27,20 @@ namespace geom_utils
 
     // Return true if polygon orientation is counter-clockwise (i.e. area is positive)
     bool Polygon::orientation() const
-    { 
-        return {}; 
+    {
```

Comment:
Looks like the `float res` here will be equal to the double area of polygon, if i'm not mistaken. So, this code(with some corrections) **should** be moved to `Polygon::area()` instead. And result of `Polygon::orientation()` method could be easily returned via `area()` call. Since you've started implementing `orientation()`, so please complete the `area()` also. You will ease a bit the work for another dev)

Reply:
is it OK, if the Polygon::area() returns negative result? or maybe it is better to left Polygon::orientation() without changes, and use this code for Polygon::area() + modification (return abs(result/2); ?

Reply:
It is totally OK: positive/negative area defines polygone orientation ccw/cw(counter clockwise/clockwise). This is widely used in different graphic libs. As a rule, graphic libs have convention: ccw polygons denote external contours and cw polygons denote holes. External contour could have hole(s) inside. Total area of group of polygons is just sum of all polygons' areas, and it works as expected even if there are holes(those will have negative area value).

Reply:
oh, i have understood. I will implement it